docs/traits.md

```diff
@@ -99,29 +99,31 @@ int main(int argc, char *argv[])
 {
   Kokkos::initialize(argc, argv);
 
-  constexpr std::size_t N = 1000;
-  std::array<float, N> a;
+  {
```

Comment:
Can you then move them a bit for clarity, so that it looks like
```c++
   Kokkos::initialize(argc, argv); 
   {

```
and
```c++

   }
   Kokkos::finalize();
```

Reply:
OK.  Applied to doc as well.